app/assets/stylesheets/subscription_first.css

```diff
@@ -0,0 +1,4 @@
+/*
+  Place all the styles related to the matching controller here.
+  They will automatically be included in application.css.
```

Comment:
@castrolem should we clean these files empty files?

Reply:
Let's leave it for now after we add styling, then we'll see which one we need and which we don't.